src/applications/vaos/tests/new-appointment/components/ConfirmationPage/ConfirmationDirectScheduleInfo.unit.spec.jsx

```diff
@@ -30,16 +41,25 @@ describe('VAOS <ConfirmationDirectScheduleInfo>', () => {
     };
     const pageTitle = 'Your appointment has been scheduled';
 
-    const tree = mount(
+    const screen = renderWithStoreAndRouter(
       <ConfirmationDirectScheduleInfo {...props} pageTitle={pageTitle} />,
+      {
+        initialState,
+        reducers,
+      },
     );
 
-    expect(tree.text()).to.contain('December 20, 2019 at 10:00 a.m. MT');
-
-    expect(tree.find('h1').text()).to.equal(pageTitle);
-
-    expect(tree.find('.vaos-u-word-break--break-word').exists()).to.be.true;
+    screen.getByText(/Your appointment has been scheduled./i);
+    screen.getByText(/December 20, 2019 at 10:00 a.m. MT/i);
+    screen.getByText(/Cheyenne VA Medical Center/i);
+    screen.getByText(/2360 East Pershing Boulevard/i);
+    screen.getByText(/Cheyenne, WY 82001-5356/i);
+    screen.getByText(/Follow-up\/Routine/i);
+    screen.getByText(/Additional info/i);
 
-    tree.unmount();
+    // Simulate user clicking the "Add to calendar" button
```

Comment:
This isn't really simulating the click, though it is good to check that the link appears with the correct name

Reply:
Done.